autoload/neomake/makers/ft/c.vim

```diff
@@ -39,3 +45,26 @@ function! neomake#makers#ft#c#gcc()
             \ '%f:%l: %m',
         \ }
 endfunction
+
+function! neomake#makers#ft#c#clangtidy()
+    " Default arguments.
+    let l:args = []
+
+    " Add user-defined arguments if some are set.
+    " The -p option followed by the path to the build directory is expected.
+    " That directory should contain the compile command database
+    " (compile_commands.json).
+    if exists("g:neomake_c_clangtidy_args_conf")
+        let l:args += g:neomake_c_clangtidy_args_conf
```

Comment:
Now that I think about it, I actually tested that when I started using neomake, but I didn't realize that `neomake_c_clangtidy_args` was not propagating to the C++ maker, and I had to set `neomake_cpp_clangtidy_args` as well.